src/fault_injector/fault_types/fault.py

```diff
@@ -34,3 +34,26 @@ def inject_fault(component: Component):
         """
 
         raise NotImplementedError()
+
+    @abstractmethod
+    def resolve_fault(component: Component):
+        """resolves the previously injected fault
+
+        :param component: the component with the injected fault
+        :type component: Component
+        """
+
+        raise NotImplementedError()
```

Comment:
```suggestion
    def resolve_fault(self, component: Component):
        """resolves the previously injected fault

        :param component: the component with the injected fault
        :type component: Component
        """

        raise NotImplementedError()
```

You probably want a reference to the fault